Example/Example/AppDelegate.swift

```diff
@@ -20,9 +20,14 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
 
         locationManager.delegate = self
         self.requestLocationPermissions()
-
+        
+        Radar.stopTracking()
+        
         // Replace with a valid test publishable key
-        Radar.initialize(publishableKey: "prj_test_pk_0000000000000000000000000000000000000000")
+        Radar.initialize(publishableKey: "prj_test_pk_0000000000000000000000000000000000000000", options: RadarInitializeOptions.fromDictionary([
+            "userId": "testUserId",
+            "metadata": [ "foo": "bar" ]
+        ]))
```

Comment:
@ShiCheng-Lu can we just make the second argument a dictionary instead of `RadarInitializeOptions` object?